src/log/log.c

```diff
@@ -12,6 +12,8 @@ static int __log_newfile(WT_SESSION_IMPL *, bool, bool *);
 static int __log_openfile(WT_SESSION_IMPL *, uint32_t, uint32_t, WT_FH **);
 static int __log_truncate(WT_SESSION_IMPL *, WT_LSN *, bool, bool);
 static int __log_write_internal(WT_SESSION_IMPL *, WT_ITEM *, WT_LSN *, uint32_t);
+static int __log_fsync_dir(WT_SESSION_IMPL *, WT_LSN *, const char *);
+static int __log_fsync_file(WT_SESSION_IMPL *, WT_LSN *, const char *, bool);
```

Comment:
If you move the functions above the uses per my comment below, you shouldn't need this at all. If you do need it, they should be alphabetized with the others.

Reply:
Done